admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+= Aggregating Logs with the EFK Stack
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Application developers can aggregate and view logs for a range of services using
+the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
+topic shows you how to deploy the stack using a template.
+
+Three components make up the EFK logging stack:
+
+* Elasticsearch: an object store where all logs are fed.
+* Fluentd: gathers logs from nodes and feeds them to Elasticsearch.
+* Kibana: a web UI for Elasticsearch.
+
+== Preperation for Deploying the EFK Stack
+
+Deploying the EFK stack requires:
+
+* A 'logging-deployer' secret
+* A 'logging-deployer' service account with the secret and create privileges
```

Comment:
Mark up the secret and SA name instead of single quotes (probably just `*bold*`).